students/Devon_Peterka/Lesson_2/Grid_Print.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+
+print('Part 1 - Explicit Grid Printing without using function')
+print('+ ' + 4*'- ' + '+ ' + 4*'- ' + '+')
+for i in range(0,4):
+    print('|' + 9*' ' + '|' + 9*' ' + '|')
+print('+ ' + 4*'- ' + '+ ' + 4*'- ' + '+')
+for i in range(0,4):
+    print('|' + 9*' ' + '|' + 9*' ' + '|')
+print('+ ' + 4*'- ' + '+ ' + 4*'- ' + '+')
+print()
+
+
+
+
+#Part 2 - Print grid of arbitrary cell size using a function
+
+def print_grid(n):
+    for i in range (0,n+2):
```

Comment:
Extra whitespace before parenthesis and need one after the comma.  Actually, need spaces after commas within all parentheses below as well.